.babelrc

```diff
@@ -6,6 +6,7 @@
   ],
   "presets": ["es2015", "stage-2"],
   "plugins": [
+    "jsx-control-statements",
```

Comment:
Yeah. I guess the only way to do this before was to create an element within JS control statements and inserting them into final element, or just splitting into multiple `return`s.

Seems like a good addition. Though tag with an uppercase letter looks a bit unusual. :P